README.md

```diff
@@ -1,70 +1,44 @@
+[partiql-specification]: https://partiql.org/assets/PartiQL-Specification.pdf
+[partiql-website]: https://www.partiql.org
+[partiql-website-plk]: https://www.partiql.org/plk/latest/
+[partiql-tests]: https://github.com/partiql/partiql-tests
+[maven-partiql-namespace]: https://central.sonatype.com/namespace/org.partiql
+[git-submodule]: https://git-scm.com/book/en/v2/Git-Tools-Submodules
 
-This is a Kotlin implementation of the [PartiQL specification](https://partiql.org/assets/PartiQL-Specification.pdf).
-PartiQL is based on SQL-99 and has added support for working with schemaless hierarchical data.
-PartiQL’s extensions to SQL are easy to understand, treat nested data as first class citizens and
-compose seamlessly with each other and SQL.
+# PartiQL Lang Kotlin
 
-This repository contains an embeddable reference interpreter, test framework, and tests for PartiQL in Kotlin.
+This is a Kotlin/JVM implementation of the [PartiQL specification][partiql-specification].
+PartiQL is based on SQL:1999 and has added support for working with schemaless hierarchical data.
+PartiQL’s extensions to SQL are easy to understand, treat nested data as first class citizens, and
+compose seamlessly with SQL.
 
 ## About
 
-Check out the [PartiQL Lang Kotlin Wiki](https://github.com/partiql/partiql-lang-kotlin/wiki) for documentation,
-tutorials, upgrade guides, and more!
+Check out the [PartiQL website][partiql-website] for documentation, usage guides, and more!
 
 ## Status
 
-Users of PartiQL should consider PartiQL to be in "preview" status. It has been leveraged within a number of Amazon internal
-systems and AWS products for over a year. The behavior of the language itself is mostly stable, however,
-the public API of the interpreter is slated to undergo significant improvements in the near term. (See the
-GitHub issues list for details.)
+Users of PartiQL should consider the PartiQL library to be stable. It has been leveraged within a number of Amazon internal
+systems and AWS products for multiple years. The behavior of the language itself is mostly stable.
 
 ## Using In Your Project
 
-This project is published to [Maven Central](https://search.maven.org/artifact/org.partiql/partiql-lang-kotlin).
-
-| Group ID      | Artifact ID           | Recommended Version |
-|---------------|-----------------------|---------------------|
-| `org.partiql` | `partiql-lang-kotlin` | `0.14.9`            |
-
-If you are looking for a pre-release version, the latest is: `1.0.0-rc.3`.
-
-For Maven builds, add the following to your `pom.xml`:
-
-```xml
-<dependency>
-  <groupId>org.partiql</groupId>
-  <artifactId>partiql-lang-kotlin</artifactId>
-  <version>${version}</version>
-</dependency>
-```
-
-For Gradle 5+, add the following to your `build.gradle`:
-
-```groovy
-repositories {
-    mavenCentral()
-}
-
-dependencies {
-    implementation "org.partiql:partiql-lang-kotlin:${version}"
-}
-```
-
-Be sure to replace `${version}` with the desired version.
```

Comment:
Perhaps could be good to still include this block with importing instructions (but swapping out `partiql-lang-kotlin` for `partiql-lang`)?